software/glasgow/applet/program/chipcon/ccdpi.py

```diff
@@ -0,0 +1,624 @@
+# Chipcon Debug and Programming Interface
+#
+# Supports all TI/Chipcon embedded 8051 radios:
+#    CC1110, CC1111, CC2510, CC2511, CC2430, CC2431, CC2530, CC2531, CC2533, CC2540, CC2541
+#
+# Base protocol from:
+#   "CC1110/CC2430/CC2510 Debug and Programming Interface Specification Rev. 1.2"
+#
+# Extended protocol from:
+#   "CC253x System-on-Chip Solution for 2.4-GHz IEEE 802.15.4 and ZigBee® Applications
+#    CC2540/41 System-on-Chip Solution for 2.4-GHz Bluetooth® low energy Applications
+#    User's Guide - Literature Number: SWRU191F April 2009–Revised April 2014"
+#
+import logging
+import asyncio
+import random
+from collections import namedtuple
+from enum import IntEnum, IntFlag
+
+from nmigen import *
```

Comment:
I had to update this to the new name to get the applet working today
```suggestion
from amaranth import *
```

Reply:
Thanks!